pkgs/by-name/zm/zmkBATx/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  qt6,
+  pkg-config,
+  dbus,
+  simpleBluez,
+  simpleDBus,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "zmkBATx";
+
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "mh4x0f";
+    repo = "zmkBATx";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-xbiwRHVTuaZDH3RZlMK2CpKBThtS8g6q5r3C+OccDZg=";
+  };
+
+  nativeBuildInputs = [
+    qt6.wrapQtAppsHook
+    pkg-config
+    qt6.qmake
+  ];
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtconnectivity
+    dbus.lib
+    simpleBluez
+    simpleDBus
+  ];
+
+  patchPhase = ''
+    runHook prePatch
+    substituteInPlace zmkBATx.pro --replace-fail "/usr/include/dbus-1.0" "${dbus.dev}/include/dbus-1.0"
+    substituteInPlace zmkBATx.pro --replace-fail "/usr/lib/x86_64-linux-gnu/dbus-1.0/include" "${dbus.lib}/lib/dbus-1.0/include"
+    substituteInPlace zmkBATx.pro --replace-fail "PREFIX = /usr/local" "PREFIX = $out"
```

Comment:
Can't we configure `PREFIX` with a configureFlag ?

Reply:
Right, done, thanks :)